src/components/Home.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Home = () => {
+  return (
+    <div className="card" style={{ margin: "10%", padding: "20px", textAlign: "center" }}>
+      <div className="row">
+        <a href="/auth/google" className="waves-effect waves-light btn deep-purple darken-2">Sign Up with Google</a>
```

Comment:
maybe change it to `/api/auth/google` to get all server calls under `/api`.
I don't know how the reverse proxy for this be set when deployed to **serverless** tho.

Thoughts @darshkpatel ?